<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongSelectWindow</class>
 <widget class="QDialog" name="SongSelectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Song</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { background-color: #222; color: white; }
QGroupBox { border: 1px solid #444; border-radius: 4px; margin-top: 20px; font-weight: bold; color: #ddd; }
QGroupBox::title { subcontrol-origin: margin; left: 10px; top: 0px; }
QListWidget { background-color: #333; border: none; color: white; font-size: 14px; }
QListWidget::item { padding: 5px; }
QListWidget::item:selected { background-color: #00AAFF; color: white; }
QTableWidget { background-color: #333; gridline-color: #444; color: white; border: none; }
QHeaderView::section { background-color: #444; color: white; padding: 4px; border: 1px solid #555; }
QPushButton { background-color: #00AAFF; color: white; border-radius: 4px; padding: 10px; font-weight: bold; font-size: 16px; }
QPushButton:hover { background-color: #0088CC; }
QPushButton:pressed { background-color: #006699; }</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupFolders">
     <property name="title">
      <string>1. Folders</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <widget class="QPushButton" name="btnScan">
        <property name="text">
         <string>Scan / Change Folder</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #444; font-size: 12px; padding: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listFolders"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSongs">
     <property name="title">
      <string>2. Songs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="listSongs"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupScore">
     <property name="title">
      <string>3. Records</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="lblBestScore">
        <property name="styleSheet">
         <string notr="true">font-size: 16px; color: gold; font-weight: bold; margin-bottom: 5px;</string>
        </property>
        <property name="text">
         <string>Best: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableHistory">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlay">
        <property name="text">
         <string>START GAME</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>